modules/claims_api/app/models/claims_api/veteran.rb

```diff
@@ -30,15 +54,26 @@ def va_profile=(new_va_profile)
       super(new_va_profile)
     end
 
+    def loa3_user
+      loa3?
+    end
+
+    def authn_context
+      'authn'
+    end
+
     def self.from_headers(headers, with_gender: false)
       veteran = new(
+        uuid: ensure_header(headers, 'X-VA-SSN'),
         ssn: ensure_header(headers, 'X-VA-SSN'),
         first_name: ensure_header(headers, 'X-VA-First-Name'),
         last_name: ensure_header(headers, 'X-VA-Last-Name'),
         va_profile: build_profile(headers),
         edipi: ensure_header(headers, 'X-VA-EDIPI'),
         last_signed_in: Time.now.utc
       )
+      # commenting this out until we decide on LOA
+      # veteran.loa = { current: 3, highest: 3 }
```

Comment:
Have you tested whether or not LoA 3 is the minimum required for this or have we tested the request with loa2?

Reply:
This is an mvi over evss requirement

Reply:
it should be passed an an option and not hard coded is all I meant.

Reply:
Since we are going to replace this with the non-veteran oauth flow. I have updated the comment in the code